tutorials/llm/llama-3/pruning-distillation/03_a_depth_pruning.ipynb

```diff
@@ -0,0 +1,77 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8bc99d2f-9ac6-40c2-b072-12b6cb7b9aca",
+   "metadata": {},
+   "source": [
+    "### Step 3: Prune the finetuned-teacher model to create a student\n",
```

Comment:
fix punctuation

Step 3: Prune the fine-tuned teacher model to create a student\n",